products/sle/main.pm

```diff
@@ -1175,6 +1181,9 @@ else {
         load_default_autoyast_tests;
         # Load this to perform some other actions before upgrade even though registration and patching is controlled by autoyast
         loadtest 'update/patch_sle';
+	if (is_sle && (get_var('FLAVOR') =~ /Migration/) && (get_var('SCC_ADDONS') !~ /ha/) && !is_sles4sap && (is_upgrade || get_var('MEDIA_UPGRADE'))) {
+            loadtest "console/check_system_info";
+        }
```

Comment:
Yutao, you used the same code triple times... how about set a function there? (IMO)

Reply:
This is main file. I need load this test modules at many places.

Reply:
That is the reason why I asked for the changes :)

Reply:
Only one module....

Reply:
yup, then no problem :)